Markdown examples update (#656)
Codecademy/docs: content/markdown/concepts/headings/headings.md

🚫 Don't: `#Breaking News`

> Put blank lines before and after a heading for compatibility and readability.

Reviewer: ```suggestion
Headings should have a blank line above and below for compatibility and readability:
```
